Atomic/diff.py

```diff
@@ -175,6 +206,17 @@ def remove(self):
         """
         self.dm.unmount()
 
+    @property
```

Comment:
I dont think so.  This will get reworked when the refactoring PR is in and comb over diff